make authmodulelist be only modules that are installed with core package
the other modules are appended to list by sed.

--- ./authdaemonrc.in~	2005-07-05 15:25:08.000000000 +0300
+++ ./authdaemonrc.in	2005-09-19 17:42:11.000000000 +0300
@@ -24,7 +24,7 @@
 # by removing them from the following list.  The available modules you
 # can use are: @ALLMODULES@
 
-authmodulelist="@ALLMODULES@"
+authmodulelist="authpam authcustom"
 
 ##NAME: authmodulelistorig:3
 #
